docs/platforms/flutter/user-feedback/index.mdx

```diff
@@ -8,12 +8,54 @@ When a user experiences an error, Sentry provides the ability to collect additio
 
 ## User Feedback API
 
-The user feedback API allows you to collect user feedback while utilizing your own UI. You can use the same programming language you have in your app to send user feedback. In this case, the SDK creates the HTTP request so you don't have to deal with posting data via HTTP.
-
-Sentry pairs the feedback with the original event, giving you additional insight into issues. Sentry needs the `eventId` to be able to associate the user feedback to the corresponding event. There are several ways to get the `eventId`:
+The user feedback API allows you to collect user feedback while utilizing your own UI. Sentry pairs the feedback with the original event, giving you additional insight into issues. Sentry needs the `associatedEventId` to be able to associate the user feedback to the corresponding event. There are several ways to get the `associatedEventId`:
 
 - use {<PlatformLink to="/configuration/options/#before-send"><PlatformIdentifier name="before-send" /></PlatformLink>}
 - use the return value of any method capturing an event.
 - use `Sentry.lastEventId` to get the ID of the last event sent.
 
 <PlatformContent includePath="user-feedback/sdk-api-example/" />
+
+## SentryFeedbackWidget
+
+Use `SentryFeedbackWidget` to let users send feedback data to Sentry.
+
+The widget requests and collects the user's name, email address, and a description of what occurred. When an event identifier is provided, Sentry pairs the feedback with the original event, giving you additional insights into issues. Additionally, you can provide a screenshot that will be sent to Sentry.
```

Comment:
I think it would make sense to also link the screenshot page because the user needs to attach `SentryWidget` to their app for screenshots to work